docs/1.0. Certificate Provisioning.md

```diff
@@ -209,19 +212,19 @@ On connection to the target environment's network the EST Client SHOULD attempt
 
 #### Get Root CA
 
-There are two possible workflows to getting the Root CA depending on if explicit trust of EST server is enabled or disabled.
+There are two possible workflows for installing the Root CA for the target network depending on if explicit trust of EST server is **Enabled** or **Disabled**.
 
 If explicit trust of the EST Server is **Enabled**, the EST Client SHOULD make a HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client SHOULD explicitly trust the EST Server manually configured or discovered using Unicast DNS and not perform authentication of the EST Server's TLS Certificate during the initial request to the EST Server.
 
 If explicit trust of the EST Server is **Disabled**, the EST Client SHOULD make a HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client MUST perform authentication of the EST Server's TLS Certificate, using the list of trusted Certificate Authorities.
 
 If the EST Server returns a HTTP 200 response, the EST Client SHOULD add the returned Root CA to the list of trusted Certificate authorities, replacing any previously installed Root CAs which were obtained via EST. It SHOULD also store any intermediate certificates returned, which are used to form the chain of trust for certificates issued by the EST CA.
 
-The EST Client MUST be able to handle the  three "Root CA Key Update" certificates OldWithOld, OldWithNew, and NewWithOld in the response, defined in Section 4.4 of [RFC 4210][RFC-4210].
+The EST Client MUST be able to handle the three "Root CA Key Update" certificates OldWithOld, OldWithNew, and NewWithOld in the response, defined in Section 4.4 of [RFC 4210][RFC-4210].
 
 #### Generate Certificate Signing(CSR) Request
 
-The EST Client SHOULD create a CSR for each digital signature algorithm it supports, with an appropriate Key Length. The CSR MUST contain a Common Name that is resolvable via DNS on the current domain and appropriate values for the other CSR fields. The manufacturer MAY allow the optional CSR fields to be configured by the operator. The EST Client MAY use an existing private key to sign the CSR, if the private key has been compromised the EST Client MUST generate a new pair to sign the CSR.
+The EST Client SHOULD create a CSR for each digital signature algorithm it supports, with an appropriate Key Length. The CSR MUST contain a Common Name and Subject Alt Name that is resolvable via DNS on the current domain and appropriate values for the other CSR fields. The EST Client MUST generate a new key pair to sign the CSR.
```

Comment:
Perhaps 'any Subject Alternate Names' as there may be more than one. Might also be worth requiring the 'serialNumber' field here, or at least recommending it. Obviously this may be harder for pure software.